ramls/item.json

```diff
@@ -233,22 +233,27 @@
           "description": "Name of the status e.g. Available, Checked out, In transit",
           "type": "string",
           "enum": [
+            "Aged to lost",
             "Available",
             "Awaiting pickup",
             "Awaiting delivery",
             "Checked out",
+            "Claimed returned",
+            "Declared lost",
             "In process",
+            "In process (non-requestable)",
             "In transit",
+            "Intellectual item",
+            "Long missing",
+            "Lost and paid",
             "Missing",
             "On order",
             "Paged",
-            "Declared lost",
+            "Restricted",
             "Order closed",
-            "Claimed returned",
+            "Unavailable",
             "Unknown",
-            "Withdrawn",
-            "Lost and paid",
-            "Aged to lost"
+            "Withdrawn"
```

Comment:
Why have these states been sorted?

Reply:
As for me, looking on that source editing manually, this is better for making source more "Human readable" hence  it would lead to allow make changes faster and to avoid of human mistakes.

Reply:
Ok, I'll resolve this question